.github/workflows/post-merge-tests-and-tag.yml

```diff
@@ -0,0 +1,26 @@
+name: Run Tests after Pull Request is merged and Tag
+
+on:
+    push:
+      branches:
+        - main
+
+jobs:
+  run-tests-and-tag:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
+      with:
+        fetch-depth: 0 
+
+    - name: Run tests
+      uses: ./.github/workflows/run-tests.yaml
+
+    - name: Tag commit if tests pass
+      if: success()
+      run: |
+        git config user.name github-actions
+        git config user.email github-actions@github.com
+        git tag -f release_test
+        git push origin release_test --force
```

Comment:
I don't think this will work as there's a branch protection rule that disallows force pushes to the main branch. Using mutable tags like this seems to be discouraged anyway, so we might need to tag with an incrementing version number like `release_test_[number]`, perhaps using the value of `github.run_id` for the number. That would avoid having to use the force option on lines 25 and 26.

Reply:
I think this may also need `permissions: contents: write` to allow Actions to push the tag to the repo. Let's talk this over tomorrow.

Reply:
Added in https://github.com/alphagov/re-request-an-aws-account/commit/a402dbfc247ee1e178f553a809d95cbd6d55dc33